config/sources/families/include/uefi_common.inc

```diff
@@ -22,27 +22,22 @@ case "${BRANCH}" in
 		;;
 
 	legacy)
-		declare -g KERNEL_MAJOR_MINOR="6.1"          # Major and minor versions of this kernel. For mainline caching.
-		declare -g KERNELBRANCH="branch:linux-6.1.y" # Branch or tag to build from. It should match MAJOR_MINOR
+		declare -g KERNEL_MAJOR_MINOR="6.1"
```

Comment:
this whole part (switch) could be moved to common / mainline, so we define this there. UEFI is not the only mainline based kernel.

Reply:
There should be "some" separation. We can't globally bump 6.1 to 6.6 if the families are not ready for it.
So this PR proposes that families associate their current to 6.1 (but nothing else), and global hooks decide which exact version of 6.1 is used.
To bump something say from 6.1 to 6.2, handling of patches etc is required, and can't be done globally.

Reply:
>  We can't globally bump 6.1 to 6.6 if the families are not ready for it.

What you have done here is nice, I am just thinking how to polish this further.

Maintaining perspective. If we bump CURRENT from previous to next LTS and 1 out of 10 will break after general patch adjustments ... we only fix/patch that one. If we are bumping per families, only those that have active maintainers will be bumped, others, which also could just work, will stay behind. I am concerned about that.

KERNEL_MAJOR_MINOR is just more generic and thus more complicated - from maintaining perspective - form of current / edge separation which in a way to identify large changes.

Reply:
That's different. "Bump" is edge from 6.6 to 6.7.
What you're mentioning is a _rollover_: move current to legacy, edge to current. That can't be done globally either, due to the .config's being version specific.

>  If we are bumping per families, only those that have active maintainers will be bumped, others, which also could just work, will stay behind. I am concerned about that.

Check @viraniac 's comments. He was already against centralized _POINT RELEASE_ management, so how would he feel about _centralized_ MAJOR/MINOR bump? We gotta leave the autonomy to bump/rollover to the family maintainers. 

The only families that can be bumped/rolled-over without the family maintainer are those that have no patches and that the .config is trivially managed. Case in point: uefi-x86 and uefi-riscv64. Everything else needs a family maintainer.

Reply:
> What you're mentioning is a rollover: move current to legacy, edge to current. That can't be done globally either, due to the .config's being version specific.

Yeah. True, that is different case.